### DISCUSSION TITLE
PS2/PS1 analog status hid messaging?

### DISCUSSION THREAD

**Discussion by NickGravley** [darthcloud/BlueRetro#211] [Q&A]
Hi, great project!  I would like to be able to report or reply the analog status of a PS2/PS1 BlueRetro system wirelessly to a generic diy gamepad I am building, based on https://github.com/lemmingDev/ESP32-BLE-Gamepad, in order to light an led on that controller...  I was able to understand some of the code in ps_spi.c (where you set and clear gpio2),  but I do not know enough about BT/HID yet to understand what functions to look for in order to try to add a fb or status message. Is this implemented in any of the non-generic controllers (SW/PS3/Wii) such that it changes an led color or something on the controller?  I have workaround in the form of adding some esp8266 for wireless serial, but I am hoping you can help me with a more integrated solution.

**Comment by darthcloud** [comment]
This is on my TODO list, the framework required for that is not present yet in BlueRetro and its not a simple thing.

 And you would need an input report with a LED usage similar to a keyboard for your BLE pad.

**Reply by NickGravley** [reply]
Thanks.
